### PR TITLE
feat: add `imToken`  to available wallet

### PR DESCRIPTION
**Pull request by izayl**
- [x] add `imToken` wallet connection
- [x] add `imToken` option to site 
- [x] add `imToken` option to README

### CODE REVIEW

File: packages/rainbowkit/src/wallets/walletConnectors/imToken/imToken.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+export const imToken = ({ chains }: ImTokenOptions): Wallet => ({
+  id: 'imToken',
+  name: 'imToken',
+  iconUrl: async () => (await import('./imToken.svg')).default,
+  iconBackground: 'transparent',
+  downloadUrls: {
+    android: 'https://play.google.com/store/apps/details?id=im.token.app',
+    ios: 'https://itunes.apple.com/us/app/imtoken2/id1384798940',
+    qrCode: 'https://token.im/',
```
Comment (nickbytes) [inline packages/rainbowkit/src/wallets/walletConnectors/imToken/imToken.ts:19]:
```suggestion
    qrCode: 'https://token.im/download',
```

@izayl other apps provide a link that will automatically pick the appropriate app store for the platform (i.e., Rainbow uses `https://rainbow.download` which takes you to the App Store on an iOS device). Any idea if imToken has one of these? In it's absence, the link I've provided might be better still.

**Comment by nickbytes** [comment]
@izayl would you mind if I added some icons that will fit better w/ our styling to this PR? Our icons should have more squared corners, and I've got some ready that I can add to this PR if that's alright with you.

**Comment by izayl** [comment]
> @izayl would you mind if I added some icons that will fit better w/ our styling to this PR? Our icons should have more squared corners, and I've got some ready that I can add to this PR if that's alright with you.

@nickbytes ok, but pls follow this brand guide: https://token.im/brand?locale=en-us

**Comment by nickbytes** [comment]
This is ready for review.

**Review by nicoglennon — APPROVED** [review]

**Pull request merged**